.github/workflows/deploy-llpkgstore.yaml

```
@@ -0,0 +1,40 @@
+name: Deploy llpkgstore to GitHub Pages
+
+on:
+    push:
+        branches: 'website'
+
+permissions:
+    pages: write
+    id-token: write
+
+jobs:
+    build-and-deploy:
+        runs-on: ubuntu-latest
+
+        strategy:
+            matrix:
+                node-version: [23.8.0]
+
+        steps:
+            - uses: actions/checkout@v3
+            - name: Use Node.js ${{ matrix.node-version }}
+              uses: actions/setup-node@v3
+              with:
+                  node-version: ${{ matrix.node-version }}
```

```suggestion
        steps:
            - name: Check out code
              uses: actions/checkout@v3
            - name: Set up Node.js
              uses: actions/setup-node@v3
              with:
                  node-version: 23.8.0
```

I see no reason to use a matrix in this job.

---

Done at 8e42fa6